lib/internal/modules/cjs/loader.js

```diff
@@ -1791,10 +1792,11 @@ function loadSource(mod, filename, formatFromNode) {
     mod[kURL] = convertCJSFilenameToURL(filename);
   }
 
+  const defaultLoad = getDefaultLoad(mod[kURL], filename);
   const loadResult = loadWithHooks(mod[kURL], mod[kFormat], /* importAttributes */ undefined,
                                    getCjsConditionsArray(),
-                                   getDefaultLoad(mod[kURL], filename));
-
+                                   defaultLoad,
+                                   validateLoadStrict);
```

Comment:
Please don't mangle

Reply:
What do you mean by mangle?

Reply:
The insane indentation (I assume it's from the atrocious autofixer)